packages/logging/src/config/env.config.ts

```diff
@@ -6,6 +6,7 @@ dotenv.config();
 
 const envSchema = z.object({
   LOG_LEVEL: z.enum(["fatal", "error", "warn", "info", "debug", "trace"]).optional().default("info"),
+  NODE_ENV: z.enum(["development", "production", "test"]).optional().default("development"),
```

Comment:
this package shouldn't depend on NODE_ENV, only on env vars specific to it and direct config passed to the constructor.

```suggestion
```